tests/lib/nested.sh

```diff
@@ -1219,13 +1221,20 @@ nested_start_core_vm_unit() {
     PARAM_RTC="${NESTED_PARAM_RTC:-}"
     PARAM_EXTRA="${NESTED_PARAM_EXTRA:-}"
 
+    local PASSPHRASE
+    PASSPHRASE=${NESTED_PASSPHRASE:-}
```

Comment:
nitpic: you do not need the `:-` part, the variable already exists due to `: "${NESTED_PASSPHRASE:=}"` at the top of the file.

Reply:
updated, thanks!